sharepoint/sharepoint-ps/sharepoint-online/Request-SPOPersonalSite.md

```diff
@@ -24,7 +24,7 @@ Request-SPOPersonalSite [-NoWait] -UserEmails <String[]> [<CommonParameters>]
 
 ## DESCRIPTION
 
-The `Request-SPOPersonalSite` cmdlet requests that the users specified be enqueued so that a Personal Site be created for each. The actual Personal site is created by a Timer Job later.
+The `Request-SPOPersonalSite` cmdlet requests that the users specified be enqueued so that a Personal Site be created for each. The actual Personal site is created by a Timer Job later. If the user already has a Personal Site created, the request for creation for the user will be silently ignored.
 
 > [!NOTE]
```

Comment:
Could we remove this !NOTE since the current version of PoSh doesn't support it?